JaeYoung/week04/49994.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Set
+
+def solution(dirs): 
+    visited: Set = set()
+    x, y = [0, 0]
+    direction: dict = {
+        "U": [0, 1],
+        "D": [0, -1],
+        "L": [-1, 0],
+        "R": [1, 0]
+    }
+    
+    for dir in dirs:  
+        adder = direction[dir]
+        nx, ny = [x + adder[0], y + adder[1]]
+        
+        if abs(nx) > 5 or abs(ny) > 5:
+            continue
+
+        x_move: List[int] = [x, nx]
+        y_move: List[int] = [y, ny]
+        x_move.sort()
+        y_move.sort()
+        move_path: List[int] = x_move + y_move
+        visited.add(tuple(move_path))
```

Comment:
```suggestion
        visited.add(tuple([x,y,nx,ny]))
        visited.add(tuple([nx,ny,x,y]))
```
이렇게 저장하시고 answer/2 하는 것도 한 번 고려해 보시면 좋을 것 같습니다.